### PR TITLE
refresh utilsobject modulvars on module install

### PR DESCRIPTION
**Pull request by keywan-ghadami-oxid**
fix ported from project, where module activation did not take affect because "moduleFiles" cache in oxUtilsObject was not refresh by moduleInstaller

### CODE REVIEW

**Comment by Sieg** [comment]
Hi @keywan-ghadami, is the bug: https://bugs.oxid-esales.com/view.php?id=5688 related?

**Comment by keywan-ghadami-oxid** [comment]
it seams to have the same root cause, but i guess this PullRequest will not fix that issue 5688 because i did only take care about activating/deactivating modules, in my case activating was done by "ModuleInstaller" Class via oxid console.

File: source/Core/ModuleInstaller.php [hunk 1]
```diff
@@ -672,5 +672,7 @@ protected function resetCache()
         if ($this->getModuleCache()) {
             $this->getModuleCache()->resetCache();
         }
+        //clear utilsObject internal cache
+        UtilsObject::resetModuleVariables();
```
Comment (robertblank) [inline source/Core/ModuleInstaller.php]: The method UtilsObject::resetModuleVariables(); is deprecated. Please use ModuleVariablesLocator::resetModuleVars instead.  See [UtilsObject.php#L382](https://github.com/OXID-eSales/oxideshop_ce/blob/64b64a29e670acd92fc2e55323df9b37ef07b64d/source/Core/UtilsObject.php#L382)

**Comment by keywan-ghadami-oxid** [comment]
@michaelkeiluweit thank you for the fix, i cherry picked your commit and removed my wrong comment within your commit.
@robertblank thank you for review, please double check if there is anything left to do on this PR.

**Comment by keywan-ghadami-oxid** [comment]
This is PR is not needed anymore because oxmodulecache already resets module variables by using modulevariablelocator.

**Pull request closed without merging**